lib/tbot/testhelpers/srv.go

```diff
@@ -37,6 +37,14 @@ import (
 	"github.com/gravitational/teleport/tool/teleport/testenv"
 )
 
+type DefaultBotConfigOpts struct {
+	// Makes the bot connect via the Auth Server instead of the Proxy server.
+	PreferAuthServerOverProxyServer bool
```

Comment:
This field feels a tad verbose in it's wording 😓 Perhaps we ought to just take the address of the remote we want the test bot to connect to ? Part of me is thinking in a new future refactor I may just remove this `DefaultBotConfig` regardless.

Reply:
I've reworked this to `UseAuthServer`. Changing this to the actual address would move the logic on how to swap the address, to all tests using the DefaultConfig. That seems a bit verbose.